core/web/cosmos_nodes_controller.go

```diff
@@ -0,0 +1,18 @@
+package web
+
+import (
+	"github.com/smartcontractkit/chainlink-cosmos/pkg/cosmos/db"
+
+	"github.com/smartcontractkit/chainlink/core/services/chainlink"
+	"github.com/smartcontractkit/chainlink/core/web/presenters"
+)
+
+// ErrCosmosNotEnabled is returned when COSMOS_ENABLED is not true.
+var ErrCosmosNotEnabled = errChainDisabled{name: "Cosmos", envVar: "COSMOS_ENABLED"}
```

Comment:
Why even bother implementing this at all? It has never done anything and will never do anything.

Reply:
Since the config is now all in toml, the chains/nodes controllers and the CLI endpoints should be removed.

Reply:
So the node controller is still used to list the nodes, which includes live state.
The chain controller was still partially functional as well, but doesn't provide any more info than the TOML IIRC.

Reply:
Yeah I think keeping things consistent for now makes sense.